src/tiled/abstractobjecttool.cpp

```diff
@@ -115,8 +116,7 @@ void AbstractObjectTool::mouseMoved(const QPointF &pos,
 void AbstractObjectTool::mousePressed(QGraphicsSceneMouseEvent *event)
 {
     if (event->button() == Qt::RightButton) {
-        showContextMenu(topMostObjectItemAt(event->scenePos()),
-                        event->screenPos());
+        showContextMenu(event->scenePos(), event->screenPos());
```

Comment:
If you meant reverting this signature change, please do because it doesn't matter in the end so keeping the patch smaller is better.